cmd/chartsvc/handler.go

```diff
@@ -407,7 +413,7 @@ func chartVersionAttributes(cid string, cv models.ChartVersion) models.ChartVers
 
 func chartAttributes(c models.Chart) models.Chart {
 	if c.RawIcon != nil {
-		c.Icon = pathPrefix + "/assets/" + c.ID + "/logo-160x160-fit.png"
+		c.Icon = pathPrefix + "/assets/" + c.ID + "/logo"
```

Comment:
I have changed the URL since the icon doesn't have to be a PNG anymore (nor be 160x160)